docs/source/filters.rst

```diff
@@ -385,6 +385,15 @@ project for the latest release version, to be notified of new releases:
      - html2text: re
      - strip
 
+This is the corresponding version for Gitlab tags:
+
+.. code:: yaml
+
+   url: https://gitlab.com/chinstrap/gammastep/-/tags
```

Comment:
Since we run automated tests on the filter documentation, you need to add the (simplified) content of this URL and the expected output to `lib/urlwatch/tests/data/filter_documentation_testdata.yaml`, just provide some simplified HTML snippet in the "input" key, and the expected output in the "output" key.

You can run the tests with "pytest -v". You can either install some dependencies that it needs (tesseract, pdftotext, ...) or just ignore those failures and check if the `lib/urlwatch/tests/test_filter_documentation.py` tests work properly.

Reply:
Only installing `pytest` somehow wasn't enough to make it run. I didn't try to figure out the dependencies, but instead repeatedly pushed edits until the checks on travis-ci passed. I hope that was ok.